lib/Conversion/TritonGPUToLLVM/Utility.cpp

```diff
@@ -289,4 +289,10 @@ Value addStringToModule(Location loc, ConversionPatternRewriter &rewriter,
 }
 
 } // namespace LLVM
+
+bool isF8(Type eType) {
+  return eType.isFloat8E5M2FNUZ() or eType.isFloat8E4M3FNUZ() or eType.isFloat8E5M2() or eType.isFloat8E5M2FNUZ();
```

Comment:
This is probably to long to fit into a single line. Can you run clang to fix the format for the files this PR has touched?

Reply:
@scxiao 
You can use `git-clang-format <commit hash>`  to reformat code to comply with `.clang-format` config of our project.

This toll will not change code which was changed before given commit, so you can reformat only your code.